ui/components/ui/box/box.js

```diff
@@ -12,26 +12,11 @@ import {
   TEXT_ALIGN,
   FLEX_DIRECTION,
   FLEX_WRAP,
+  BREAKPOINTS,
 } from '../../../helpers/constants/design-system';
 
-const ValidSize = PropTypes.oneOf([
-  0,
-  1,
-  2,
-  3,
-  4,
-  5,
-  6,
-  7,
-  8,
-  9,
-  10,
-  11,
-  12,
-  'auto',
-]);
-
-export const ValidBackgroundColors = [
+const Sizes = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+export const BackgroundColors = [
```

Comment:
Differentiating values and prop type validators by using the prefix `Valid` for PropTypes. `BackgroundColors` and `BorderColors` are used in the story